app/scripts/views/sign_up.js

```diff
@@ -38,20 +38,38 @@ function (_, p, BaseView, FormView, Template, Session, AuthErrors,
     date: now.getDate()
   };
 
+  var EMAIL_SELECTOR = 'input[type=email]';
+
   var View = FormView.extend({
     template: Template,
     className: 'sign-up',
 
     beforeRender: function () {
-      if (document.cookie.indexOf('tooyoung') > -1) {
-        this.navigate('cannot_create_account');
-        return p(false);
-      }
-
-      // TODO #1913 - get from the User model when ready.
-      this._bouncedEmail = this.ephemeralMessages.get('bouncedEmail');
+      var self = this;
+      return p().then(function () {
+        if (document.cookie.indexOf('tooyoung') > -1) {
+          self.navigate('cannot_create_account');
+          return false;
+        }
+
+        // TODO #1913 - get from the User model when ready.
```

Comment:
@ckarlof suggested at one point that we stuff the `bouncedEmail` into the `User` or `Account` model to avoid stuffing user/account related data in random locations.